source/Img32.pas

```diff
@@ -557,6 +557,8 @@   TImageList32 = class
   MulTable: array [Byte,Byte] of Byte;
   //DivTable[a,b] = a * 255/b (for a &lt;= b)
   DivTable: array [Byte,Byte] of Byte;
+  // DivOneByX[x] = 1/x
+  DivOneByXTable: array[Word] of Double;
```

Comment:
ISTM that the DivOneBy table would be better located in `Img32.Transform` since it adds a very small overhead to loading the library but only be used when `Img32.Transform is used.

Reply:
It is now in Img32.Transform.